src/components/sections/FeaturedPools.vue

```diff
@@ -3,55 +3,61 @@
     <h3 class="mb-4">Featured ecosystem protocol</h3>
     <div class="grid grid-cols-12 gap-x-4">
       <div class="mb-4 col-span-12 lg:col-span-6">
-        <BalFeatureCard imgSrc="/images/banners/element.png">
-          <div class="flex flex-col justify-between h-full">
-            <div>
-              <h4 class="mb-2">Element Finance</h4>
-              <p class="mb-4">
-                Element is an open source protocol for fixed and variable yield
-                markets. It is built on Balancer V2. View and manage your
-                liquidity positions directly on Element.fi.
-              </p>
-            </div>
+        <BalCard
+          title="Element Finance"
+          imgSrc="/images/banners/element.png"
+          hFull
+          growContent
+        >
+          <p>
+            Element is an open source protocol for fixed and variable yield
+            markets. It is built on Balancer V2. View and manage your liquidity
+            positions directly on Element.fi.
+          </p>
+          <template v-slot:footer>
```

Comment:
In general BalCard should be able to handle this kind of layout / card style with very little customisation.